subjects/mobile-dev/stock-market/resources/mock-stock-data-server/utils.py

```diff
@@ -0,0 +1,25 @@
+import os
+import csv
+
+
+def load_historical_data(directory_path='./sample-stocks/'):
+    historical_data = {}
+
+    file_list = [filename for filename in os.listdir(directory_path) if filename.endswith(".csv")]
+    for filename in file_list:
+        if filename.endswith(".csv"):
```

Comment:
Unnecessary if statement here, the check is already done during the initialization of file_list using list comprehension [Line 8]

Reply:
True :)